WeakAurasTemplates/TriggerTemplatesDataBCC.lua

```diff
@@ -110,11 +110,11 @@ templates.class.WARRIOR = {
         { spell = 7384, type = "ability", requiresTarget = true, form = 1}, -- Overpower
         { spell = 7386, type = "ability", requiresTarget = true, debuff = true}, -- Sunder Armor
         { spell = 12323, type = "ability", debuff = true, talent = 46}, -- Piercing Howl
-        { spell = 12292, type = "ability", buff = true, talent = 53}, -- Sweeping Strikes
+        { spell = 12328, type = "ability", buff = true, talent = 53}, -- Sweeping Strikes
```

Comment:
Eh the old thing looks correct to me:
13292 is sweeping strikes, and it's talent number 1, 13
123228 is death wish and it's talent number 0, 13

What is going on?

Reply:
Those 2 talents were inverted between Arms branch & Fury Branch on BCC
![image](https://user-images.githubusercontent.com/189656/124386510-231d3f80-dcdb-11eb-8424-84767058f9f5.png)
![image](https://user-images.githubusercontent.com/189656/124386520-303a2e80-dcdb-11eb-8bf4-dceb981c7426.png)

Reply:
I don't understand what is currently wrong:

Death Wish, spell id: 12328 https://classic.wowhead.com/spell=12328/death-wish is talent 0 + 13 = 13
Sweeping Strikes, spell id:  12292 https://classic.wowhead.com/spell=12292/sweeping-strikes is talent 1 +13 = 53

The switch is in the BCC data compared to vanilla classic.

Reply:
<https://tbc.wowhead.com/spell=12328/>
<https://tbc.wowhead.com/spell=12292>

Reply:
Oh wow.